ggshield/cmd/iac/scan.py

```diff
@@ -78,6 +79,7 @@ def scan_cmd(
     update_context(
         ctx, exit_zero, minimum_severity, verbose, ignore_policies, ignore_paths, json
     )
+
```

Comment:
Nit: avoid commiting meaningless changes.

Reply:
Yes sorry, the PR was not ready...